scripts/is-it-pretty.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+node_modules/.bin/prettier-standard '**/*.js'
+if [ -z "$(git status --porcelain)" ]; then
+  # Working directory clean
+  echo 'OK'
+  exit 0
+else
+  # Uncommitted changes
+  echo 'BAD'
+  exit 1
+fi
```

Comment:
great change, but we should have more descriptive error messages than "OK" and "BAD"

Reply:
@victorsingh if NO errors, print nothing, just exit 0, if there are errors, then print a more informative message